rafthttp/pipeline_test.go

```diff
@@ -77,7 +79,7 @@ func TestPipelineExceedMaximalServing(t *testing.T) {
 	}
 
 	// unblock the senders and force them to send out the data
-	tr.unblock()
+	tr.CancelRequest(&http.Request{})
```

Comment:
can we change this back to unblock for better readability? we want to have a tr that can blocks the requests until we unblock it. After the unblock, it should be able to accept and send requests in the future.